applications/StructuralMechanicsApplication/python_scripts/set_automated_initial_variable_process.py

```diff
@@ -0,0 +1,72 @@
+# Importing the Kratos Library
+from genericpath import isfile
+import KratosMultiphysics as KM
+import KratosMultiphysics.StructuralMechanicsApplication as SMA
+from KratosMultiphysics import Logger
+from KratosMultiphysics.read_csv_table_utility import ReadCsvTableUtility
+
+def Factory(settings, Model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("Expected input shall be a Parameters object, encapsulating a json string")
+
+    default_settings = KM.Parameters(
+        """{
+            "help"                     : "This automates the application of initial strain/stress variables using csv tables",
+            "model_part_name"          : "please_specify_model_part_name",
+            "variable_name"            : "SPECIFY_VARIABLE_NAME",
+            "hole_generatrix_axis"     : [0.0,0.0,1.0],
+            "hole_generatrix_point"    : [0.0,0.0,0.0],
+            "hole_radius_offset"       : 0.0,
+            "last_layer"               : false,
+            "initial_variable_table"     : {
+                        "name"             : "csv_table",
+                        "filename"         : "sample.csv",
+                        "delimiter"        : ",",
+                        "skiprows"         : 1,
+                        "first_column_id"  : 0,
+                        "second_column_id" : 1,
+                        "table_id"         : -1,
+                        "na_replace"       : 0.0
```

Comment:
these parameters are duplicated in C++

please separate them and only put what you are using